custom_components/plugwise/climate.py

```diff
@@ -292,7 +285,7 @@ async def async_set_hvac_mode(self, hvac_mode: str) -> None:
         if hvac_mode != HVACMode.OFF:
             await self.coordinator.api.set_schedule_state(
                 self.device[LOCATION],
-                STATE_ON if hvac_mode == HVACMode.AUTO else STATE_OFF,
+                STATE_ON if hvac_mode == HVACMode.AUTO else HVACMode.OFF,
```

Comment:
Sorry, I don't like this. Better to keep the schedule state separated from HVACModes.